scripts/sql/postgre/apolloportaldb.sql

```diff
@@ -0,0 +1,379 @@
+    "DataChange_LastModifiedBy" varchar(64)          DEFAULT '',                -- COMMENT '最后修改人邮箱前缀'
+    "DataChange_LastTime"       timestamp NULL DEFAULT CURRENT_TIMESTAMP,       -- COMMENT '最后修改时间'
+    CONSTRAINT "UserRole_Id" PRIMARY KEY ("Id"),
+    CONSTRAINT "UK_UserId_RoleId_DeletedAt" UNIQUE ("UserId", "RoleId", "DeletedAt")
+);
+
+CREATE INDEX "UserRole_IX_DataChange_LastTime" ON apolloportal."UserRole" ("DataChange_LastTime");
+CREATE INDEX "UserRole_IX_RoleId" ON apolloportal."UserRole" ("RoleId");
+
+/* # Dump of table Users */
+/* # ------------------------------------------------------------ */
+
+DROP TABLE IF EXISTS apolloportal."Users";
+
+CREATE TABLE apolloportal."Users"
+(
+    "Id"              BIGSERIAL,                               --  COMMENT '自增Id'
+    "Username"        varchar(64)  NOT NULL DEFAULT 'default', -- COMMENT '用户登录账户'
+    "Password"        varchar(512) NOT NULL DEFAULT 'default', -- COMMENT '密码'
+    "UserDisplayName" varchar(512) NOT NULL DEFAULT 'default', -- COMMENT '用户名称'
+    "Email"           varchar(64)  NOT NULL DEFAULT 'default', -- COMMENT '邮箱地址'
+    "Enabled"         SMALLINT              DEFAULT NULL,      -- COMMENT '是否有效'
+    CONSTRAINT "Users_Id" PRIMARY KEY ("Id"),
+    CONSTRAINT "UK_Username" UNIQUE ("Username")
+);
+
+
+/* # Dump of table Authorities */
+/* # ------------------------------------------------------------ */
+
+DROP TABLE IF EXISTS apolloportal."Authorities";
+
+CREATE TABLE apolloportal."Authorities"
+(
+    "Id"        BIGSERIAL,
+    "Username"  varchar(64) NOT NULL,
+    "Authority" varchar(50) NOT NULL,
+    CONSTRAINT "Authorities_Id" PRIMARY KEY ("Id")
+);
+
+
+/* # Config */
+/* # ------------------------------------------------------------ */
+INSERT INTO apolloportal."ServerConfig" ("Key", "Value", "Comment")
+VALUES ('apollo.portal.envs', 'dev', '可支持的环境列表'),
+       ('organizations',
+        '[{"orgId":"TEST1","orgName":"样例部门1"},{"orgId":"TEST2","orgName":"样例部门2"}]',
+        '部门列表'),
+       ('superAdmin', 'apollo', 'Portal超级管理员'),
+       ('api.readTimeout', '10000', 'http接口read timeout'),
+       ('consumer.token.salt', 'someSalt', 'consumer token salt'),
+       ('admin.createPrivateNamespace.switch', 'true', '是否允许项目管理员创建私有namespace'),
+       ('configView.memberOnly.envs', 'pro', '只对项目成员显示配置信息的环境列表，多个env以英文逗号分隔'),
+       ('apollo.portal.meta.servers', '{}', '各环境Meta Service列表');
+
+
+INSERT INTO apolloportal."Users" ("Username", "Password", "UserDisplayName", "Email", "Enabled")
+VALUES ('apollo', '$2a$10$7r20uS.BQ9uBpf3Baj3uQOZvMVvB1RN3PYoKE94gtz2.WAOuiiwXS', 'apollo', 'apollo@acme.com', 1);
+
+INSERT INTO apolloportal."Authorities" ("Username", "Authority")
+VALUES ('apollo', 'ROLE_user');
+
+-- spring session (https://github.com/spring-projects/spring-session/blob/main/spring-session-jdbc/src/main/resources/org/springframework/session/jdbc/schema-postgresql.sql)
+CREATE TABLE apolloportal.SPRING_SESSION
```

Comment:
Is it better to quote the table name just like we did for other tables?

Reply:
It's copied from spring-session repo. I suggest we not change this. And spring-session connect to lower-case table.